setup.py

```diff
@@ -193,7 +185,7 @@ def version(self):
 setup_args = dict(
     name                = 'Shapely',
     version             = str(shapely_version),
-    requires            = ['Python (>=2.7)', 'libgeos_c (>=3.3)'],
+    requires            = ['Python (>=3.5)', 'libgeos_c (>=3.3)'],
```

Comment:
Should probably add a "python_requires" field to the setup args (see https://packaging.python.org/guides/distributing-packages-using-setuptools/#python-requires), that will make pip not try to install it it (and run into the runtime error defined above)

Reply:
I've now added this, but only if we are using setuptools' setup(). The kwargs are different than for distutils' setup(). I'll explain further in the main thread ...